app/controllers/application_controller.rb

```diff
@@ -31,10 +36,13 @@ def get_tag_results(tag_string, after, before, link, limit = 15)
         track :error, :e621_posts_api_fail, response: response
         return nil
       end
-
+  
       results = JSON.parse(response.body)['posts']
-
       if results.present? && results.class == Array
+        if /order:random/i !~ padded_tag_string
+          Rails.cache.write(key, results, expires_in: 45.minutes)
+        end
```

Comment:
I'd prefer to still cache random posts, if only for a short amount of time.

Reason being, a lot of forks have no changes in their blacklist from the parent, so a 1 minute cache would be ideal just to push thru any bot sets that hit a fork.

Last weeks botting mess saw me hammering e621 pretty badly, want to limit things where I can. (Expo backoff coming... uh... soon?)

```suggestion
        if /order:random/i =~ padded_tag_string
          Rails.cache.write(key, results, expires_in: 45.minutes)
        else
          Rails.cache.write(key, results, expires_in: 1.minute)
        end
```

Reply:
Could this situation ever arise? The only time `order:random` is legal is when it comes from porn bots, and they run every 10 minutes.
Better safe than banned I guess :3c